src/components/Accordion/AccordionSection.test.js

```diff
@@ -1,95 +1,86 @@
 import React from 'react'
-import { mount } from 'enzyme'
+import { render } from '@jestutils'
+import '@testing-library/jest-dom'
 import AccordionSection from '@components/Accordion/AccordionSection'
 
 const props = {
   handleOnClick: jest.fn(),
-  label: 'Test Label'
+  label: 'Test Label',
+  children: 'I am just children element'
 }
-const component = <AccordionSection {...props} />
 
 describe('<AccordionSection /> mount', () => {
-  let wrapper
-
-  beforeEach(() => {
-    wrapper = mount(component)
-  })
-
-  afterEach(() => {
-    wrapper.unmount()
-  })
-
   it('is mounted', () => {
-    expect(wrapper.exists()).toBe(true)
+    render(<AccordionSection {...props} />)
   })
 
   it('calls function passed as handleOnClick prop on click event', () => {
-    wrapper.find('div.accordion__header').simulate('click')
+    const { getByTestId } = render(<AccordionSection {...props} />)
+
+    const header = getByTestId('accordion-header')
 
-    expect(wrapper.prop('handleOnClick')).toHaveBeenCalled()
+    header.click()
```

Comment:
Przydałoby się sprawdzić to jak wyżej, czy funkcja została wywołana` toHaveBeenCalled` pokrycie testami jest, ale tak na prawdę ten test zawsze przejdzie bo nie sprawdzamy wywołania tej funkcji